tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,33 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things
+// such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a process running on an individual node.
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	cpuUsageVar = 100
+)
+
+// This func checks the cpu usage of the cluster agent. If the usage is too high the func will return a warning.
+func CheckCPU(node *nodes.Node) (string, error) {
+	command := "ps -C agent -o %cpu --no-header"
+	output, err := node.ExecuteCommand(command)
+	str_output := output[:strings.IndexByte(output, '\n')]
+	if err != nil {
+		return output, err
+	}
+
+	output_int, err := strconv.ParseFloat(strings.TrimSpace(str_output), 32)
+	if output_int > cpuUsageVar {
+		logrus.Infof("WARNING: cluster agent cpu usage is too high. Current cpu usage is: " + str_output)
```

Comment:
If we just want to log the warning. Please do` logrus.Warn`. I believe that's the call.

Reply:
Also, do we want `WARNING` to be `stylized` as `warning`? I believe that's best practice with Go, but could be wrong.